include/roboteam_ai/coach/defence/DefenceDealer.h

```diff
@@ -15,7 +15,7 @@ namespace coach {
 ///This class keeps track of what all the defenders are doing and assigns them and communicates with them
 class DefenceDealer {
     private:
-        const int LOCKTIME=18;
+        int LOCKTIME=18;
```

Comment:
I don't see why it cannot be const. (Also, if you can edit the variable you should make it lowercase)